.azure-pipelines/generation-templates/capture-metadata.yml

```
@@ -97,11 +97,7 @@ steps:
 - template: use-dotnet-sdk.yml
 
 # verify that generated metadata is parsable as an Edm model
-- task: DotNetCoreCLI@2
-  inputs:
-    command: 'run'
-    projects: $(Build.SourcesDirectory)/msgraph-metadata/tools/MetadataParser/MetadataParser.csproj
-    arguments: ${{ parameters.cleanMetadataFile }}
+- pwsh: dotnet run --project $(Build.SourcesDirectory)/msgraph-metadata/tools/MetadataParser/MetadataParser.csproj -- ${{ parameters.cleanMetadataFile }}
```

```suggestion
- pwsh: dotnet run --project $(Build.SourcesDirectory)/msgraph-metadata/tools/MetadataParser/MetadataParser.csproj ${{ parameters.cleanMetadataFile }}
```
Will the extra -- cause any issues?

---

no, that's the syntax to pass arguments to the child process